primitives/types/extra_test.go

```diff
@@ -7,10 +7,32 @@ import (
 	"testing"
 
 	sc "github.com/LimeChain/goscale"
+	"github.com/LimeChain/gosemble/constants/metadata"
 	"github.com/stretchr/testify/assert"
 )
 
 var (
+	metadataIds = map[string]int{
+		"Bool":   metadata.PrimitiveTypesBool,
+		"String": metadata.PrimitiveTypesString,
+		"U8":     metadata.PrimitiveTypesU8,
+		"U16":    metadata.PrimitiveTypesU16,
+		"U32":    metadata.PrimitiveTypesU32,
+		"U64":    metadata.PrimitiveTypesU64,
+		"U128":   metadata.PrimitiveTypesU128,
+		"U256":   metadata.PrimitiveTypesU256,
+		"I8":     metadata.PrimitiveTypesI8,
+		"I16":    metadata.PrimitiveTypesI16,
+		"I32":    metadata.PrimitiveTypesI32,
+		"I64":    metadata.PrimitiveTypesI64,
+		"I128":   metadata.PrimitiveTypesI128,
+		"H256":   metadata.TypesH256,
```

Comment:
Can we extract these in a static func and call it where needed? There are a few places where the map consists of the same values.

Reply:
I tried this but since those maps are in a **var** statement, I'm unable to add more key-values to a map after I initialize it in the 'var' statement using a static function. I get the error '<map_name> redeclared in this package'.